config/routes.rb

```diff
@@ -0,0 +1,6 @@
+Rails.application.routes.draw do
+  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
+  root "greetings#random"
```

Comment:
- [ ] I see you set the root for the random greetings but you don't have a greetings folder in your view and as a result, it won't work.
- [ ] Also, you need to set the route for the API so that it can also be accessed.

Reply:
Suggested changes implemented